### ISSUE TITLE
Installation gives: can't load package: package code.gitea.io/tea: cannot find package "code.gitea.io/tea"

### ISSUE THREAD

**Issue by LittleColin** [go-gitea/tea#16]
Whilst I nearly had tea working before, I've rebuilt my laptop, reinstalled golang and followed the readme. However when I run 
`go install code.gitea.io/tea`

I get:
```
can't load package: package code.gitea.io/tea: cannot find package "code.gitea.io/tea" in any of:
        C:\Go\src\code.gitea.io\tea (from $GOROOT)
        C:\Users\(me)\go\src\code.gitea.io\tea (from $GOPATH)
```

**Comment by lunny** [comment]
Just `mkdir $GOPATH/src/code.gitea.io`and `cd $GOPATH/src/code.gitea.io && git clone git@github.com:go-gitea/tea.git`

**Comment by root360-AndreasUlm** [comment]
@LittleColin you can also run `go get code.gitea.io/tea` first. This command will download the required source code.

**Comment by biocyberman** [comment]
`go get code.gitea.io/tea` is the way to go. The `git clone` approach still faces problem with missing dependencies.

**Comment by lafriks** [comment]
instructions should probably need to be updated

**Comment by lunny** [comment]
resolved by #19
